crates/cargo-contract/src/cmd/mod.rs

```diff
@@ -310,6 +314,36 @@ where
     Ok(arr.into())
 }
 
+/// Prompt the user to confirm the upload of unverifiable code to the production chain.
+pub fn prompt_confirm_unverifiable_upload(chain: &str) -> Result<()> {
+    println!(
+        "{} (skip with --skip-validate)",
+        "Confirm upload:".bright_white().bold()
+    );
+    let warning = format!(
+        "You are trying to upload unverifiable code to {} mainnet",
+        chain
+    )
+    .bold()
+    .yellow();
+    print!("{}", warning);
+    println!(
+        "{} ({}): ",
+        "\nContinue?".bright_white().bold(),
+        "Y/n".bright_white().bold()
+    );
+
+    let mut buf = String::new();
+    io::stdout().flush()?;
+    io::stdin().read_line(&mut buf)?;
+    match buf.trim().to_lowercase().as_str() {
+        // default is 'y'
```

Comment:
I am not sure if the default should be 'yes'; rather, we should recommend uploading only verifiable contracts to production chains. WDYT?

Reply:
We do advise it, but we still want to give a user an option to upload an unverifiable contract (aka shoot their own leg)

Reply:
I was just thinking to make `n` as default